google-cloud-identity_toolkit-v2/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "identitytoolkit.googleapis.com",
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-identity_toolkit-v2/latest",
+    "distribution_name": "google-cloud-identity_toolkit-v2",
+    "language": "ruby",
+    "name": "identitytoolkit",
+    "name_pretty": "Identity Platform V2 API",
+    "product_documentation": "https://cloud.google.com/identity-platform/",
```

Comment:
I think this might be a more useful landing page: https://cloud.google.com/identity-platform/docs/reference/rest